Sources/Rendering/Core/VolumeProperty/Constants.d.ts

```diff
@@ -9,8 +9,13 @@ export declare enum OpacityMode {
 	PROPORTIONAL = 1,
 }
 
+export declare enum ColorMixPreset {
+	ADDITING = 0,
```

Comment:
Calling this `ADDITIVE` would be more appropriate and more in-line with the volume mapper blend modes.